accel/tcg/user-exec.c

```diff
@@ -274,7 +274,12 @@ int cpu_signal_handler(int host_signum, void *pinfo,
 
     pc = PC_sig(uc);
     return handle_cpu_signal(pc, info,
-                             TRAP_sig(uc) == 0xe ? (ERROR_sig(uc) >> 1) & 1 : 0,
+#if defined(__FreeBSD__) || defined(__DragonFly__)
+                             TRAP_sig(uc) == 0xc
```

Comment:
Why not use T_PAGEFLT?

Reply:
Mostly because we share this block with Dragonfly and I don't have a way to test there. If we don't care about keeping these changes buildable there then it should be fine to use the named constant.

Reply:
Right now, we only care about FreeBSD. However, this is in the qemu equivalent of MI code, so maybe this approach is right.